HostDependencies.md

```diff
@@ -66,17 +101,16 @@ sudo sh -c "echo 'deb http://ftp.debian.org/debian stretch-backports main' > /et
 ```
 Then install `cmake` with
 ```
-sudo apt-get update
-sudo apt-get -t stretch-backports install cmake
+sudo apt update
+sudo apt -t stretch-backports install cmake
 ```
 
 ### Python Dependencies
 
 Regardless of your Linux distribution, python dependencies are required to build seL4, the manual and its proofs. To install you can run:
 
 ```
-pip install --user setuptools
-pip install --user sel4-deps
+pip install --user setuptools sel4-deps
```

Comment:
We've found in our Dockerfiles that sometimes `pip` needs to have `setuptools` before it can do anything else, so leaving these separated would be best.

Reply:
I seperate those.